internal/configurations/configurations.go

```diff
@@ -124,13 +118,11 @@ func CreateConfiguration(ctx context.Context, cluster *kubernetes.Cluster, name,
 		sdata[k] = []byte(v)
 	}
 
-	err = cluster.CreateLabeledSecret(ctx, namespace, secretName, sdata,
+	err = cluster.CreateLabeledSecret(ctx, namespace, name, sdata,
 		map[string]string{
-			// "epinio.suse.org/configuration-type": "custom",
-			"epinio.suse.org/configuration": name,
-			"epinio.suse.org/namespace":     namespace,
-			"app.kubernetes.io/name":        "epinio",
```

Comment:
I believe that this label should be kept, as a means of declaring (to the world) that this configuration was created by the epinio server.

Reply:
ok fixing.